packages/account/src/Sections/Profile/PhoneVerification/resend-code-timer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Button, CaptionText } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
```

Comment:
same here about translations 🙏

Reply:
We will move it as a improvement card. Will not block the release because of changing the files to @deriv-com/translation